tests/gtest/avifreadimagetest.cc

```diff
@@ -370,6 +370,88 @@ TEST(ICCTest, RGB2Gray2RGB) {
   }
 }
 
+TEST(GrayTest, Roundtrip) {
+  constexpr char file_name[] = "paris_icc_exif_xmp.png";
+  const std::string file_path = std::string(data_path) + file_name;
+
+  // Read the ground truth image as gray.
+  ImagePtr image(avifImageCreateEmpty());
+  ASSERT_NE(image, nullptr);
+  ASSERT_NE(avifReadImageForRGB2Gray2RGB(file_path, AVIF_PIXEL_FORMAT_YUV400,
```

Comment:
Unrelated to this change, but it's a little strange that `avifReadImageForRGB2Gray2RGB` is defined locally here, but looks like it's part of the general library by its name. (I had to check because I was wondering about the parameter name used in the comment.)

Reply:
Ack, will fix in a later PR